src/lib/funding.js

```diff
@@ -113,3 +114,9 @@ export function precacheRememberedFunding() : ZalgoPromise<void> {
     }
     return ZalgoPromise.resolve();
 }
+
+export function getRenderedButtons(props : Object) : string {
+
+    const { sources } = normalizeProps(props);
+    return sources;
```

Comment:
CI is failing with the following type error:
```

Cannot return `sources` because `FundingList` [1] is incompatible with string [2].

   src/lib/funding.js:121:12
   121|     return sources;
                   ^^^^^^^

References:
   src/button/props.js:50:15
    50|     sources : FundingList,
                      ^^^^^^^^^^^ [1]
   src/lib/funding.js:118:54
   118| export function getRenderedButtons(props : Object) : string {
                                                             ^^^^^^ [2]
```

You may need to use `sources.toString()`